app/views/pets/show.html.erb

```diff
@@ -6,6 +6,6 @@
 <p>Shelter: <%= @pet.shelter.name %></p>
 <p>Adoption status: <%= "adoptable" if @pet.adoptable %></p>
 <ul>
-  <li><%= link_to "Update Pet", "/pets/#{@pet.id}", method: :delete %></li>
```

Comment:
This looks like it was just maybe copied and pasted from the delete link. My change made it pass the test, but I'm not sure the changes I made are the best possible version. I also may have misunderstood what you were trying to do here. Please feel free to correct me if necessary.

Reply:
Oy, I am a bit embarrassed.  This was the silly error I made with `"Update Pet"` that I told you about during our DTR, where I had copied and pasted code to create a link to delete a pet and I forgot to remove the `method: :delete`.  I must not have pushed the fixed code up to github.  Thankfully, it's an easy fix.

Reply:
You got it!  Thanks for fixing this 👍